code/modules/mob/inventory.dm

```diff
@@ -295,6 +295,8 @@ var/list/slot_equipment_priority = list( \
 			var/turf/end_T = get_turf(target)
 			if(start_T && end_T)
 				var/mob/M = item
+				if(disabilities & PACIFIST)
+					to_chat(src, "<span class='notice'>You gently let go of [M].</span>")

```

Comment:
Is this supposed to drop the mob?

Reply:
No, that's handled down south. This just adds a message.